docs/downloads/automation-library/standard/summarize_language_changes.cm

```diff
@@ -0,0 +1,44 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+
+automations:
+  {% for item in labels %}
+  summarize_language_changes_{{ item.name }}:
+      if:
+        - {{ files | match(regex=item.resources) | some }}
+      run:
+        - action: add-label@v1
+          args:
+            label: '{{ item.additions | round }}% {{ item.name }}'
+  {% endfor %}
+
+labels:
```

Comment:
Instead of using labels, this should post a comment that lists each language and shows the percent of changes. Take a look at this configuration to see an example of how that can work: https://github.com/gradle/gradle/blob/master/.cm/summary_table.cm#L39

Make sure you update the image when you make this change.